### DISCUSSION TITLE
Update the "Builds & Examples" area on documentation

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#881] [General]
As the project grows, it would be very helpful for a lot of users to see some ideas/examples of currently supported setups, like for example using JK-BMS and other options recently added.

Hope this makes any sense. Cheers and thank you for this awesome piece of work!

**Comment by swingstate** [comment]
Do you have a build to share?

**Reply by greymda** [reply]
working on and definitely will be sharing, it will take a while- just ordering all the bit and bobs currently.
